spec/pundit_spec.rb

```diff
@@ -25,6 +25,14 @@
       expect(Pundit.authorize(user, post, :update?)).to be_truthy
     end
 
+    it "returns the record on successful authorization" do
+      expect(Pundit.authorize(user, post, :update?)).to be(post)
+    end
+
+    it "returns the record when passed record with namespace " do
+      expect(Pundit.authorize(user, [:project, comment], :update?)).to be(comment)
+    end
```

Comment:
For completeness, is it worth adding another spec for the nested namespace case (`[:foo, :bar, Post]`)?https://github.com/varvet/pundit#policy-namespacing